Automattic/jetpack
# Add endpoints whitelisting for media

Fixes #

## Proposed changes:
Added 2 new endpoint whitelistings

1. for the new media endpoint (so that we can fetch images from the site)
2. for the openverse media fetch


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:







You need to test locally by using the branch `origin/feature/new-image-sources` for widget to load the new media window
![Screenshot 2023-10-13 at 17 16 31](https://github.com/Automattic/jetpack/assets/1416426/a2019fc4-03d7-4dc9-950f-cdc29fd10489)
![Screenshot 2023-10-13 at 17 10 03](https://github.com/Automattic/jetpack/assets/1416426/23fee09d-056a-4fb1-949c-733195967632)

ensure that you can open the media and that the endpoints work
